en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md

```diff
@@ -0,0 +1,1282 @@
+        PT_RESOURCE_ID INTEGER                NOT NULL,
+        PT_ID          INTEGER                NOT NULL,
+        PRIMARY KEY (ID),
+        FOREIGN KEY (PT_ID) REFERENCES IDN_UMA_PERMISSION_TICKET (ID) ON DELETE CASCADE,
+        FOREIGN KEY (PT_RESOURCE_ID) REFERENCES IDN_UMA_RESOURCE (ID) ON DELETE CASCADE
+        );
+        
+        DROP TABLE IF EXISTS IDN_UMA_PT_RESOURCE_SCOPE;
+        DROP SEQUENCE IF EXISTS IDN_UMA_PT_RESOURCE_SCOPE_SEQ;
+        CREATE SEQUENCE IDN_UMA_PT_RESOURCE_SCOPE_SEQ;
+        CREATE TABLE IDN_UMA_PT_RESOURCE_SCOPE (
+        ID             INTEGER DEFAULT NEXTVAL ('IDN_UMA_PT_RESOURCE_SCOPE_SEQ') NOT NULL,
+        PT_RESOURCE_ID INTEGER                NOT NULL,
+        PT_SCOPE_ID    INTEGER                NOT NULL,
+        PRIMARY KEY (ID),
+        FOREIGN KEY (PT_RESOURCE_ID) REFERENCES IDN_UMA_PT_RESOURCE (ID) ON DELETE CASCADE,
+        FOREIGN KEY (PT_SCOPE_ID) REFERENCES IDN_UMA_RESOURCE_SCOPE (ID) ON DELETE CASCADE
+        );
+        ```
+
+    
+3.  Download the identity component migration resources and unzip it in a local directory.
+
+    Navigate to the [latest release tag](https://github.com/wso2-extensions/apim-identity-migration-resources/releases) and download the `wso2is-migration-x.x.x.zip` under Assets.
+
+    Let's refer to this directory that you downloaded and extracted as `<IS_MIGRATION_TOOL_HOME>`.
+
+4.  Copy the `migration-resources` folder from the extracted folder to the `<API-M_4.1.0_HOME>` directory.
+
+5.  Open the `migration-config.yaml` file in the migration-resources directory and make sure that the `currentVersion` element is set to 5.7.0, as shown below.
+
+    ``` java
+    migrationEnable: "true"
+    currentVersion: "5.7.0"
+    migrateVersion: "5.11.0"
+    ```
+
+6. Remove the following 3 steps from  migration-config.yaml which is included under version: "5.10.0"
+    ```
+    -
+        name: "MigrationValidator"
+        order: 2
+    -
+        name: "SchemaMigrator"
+        order: 5
+        parameters:
+        location: "step2"
+        schema: "identity"
+    -
+        name: "TenantPortalMigrator"
+        order: 11   
+    ```
+
+7. Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.
+
+8. Update <API-M_4.1.0_HOME>/repository/conf/deployment.toml file as follows, to point to the previous user store.
+
+    ```
+    [user_store]
+    type = "database"
+    ```
+    
+9.  Start WSO2 API Manager 4.1.0 as follows to carry out the complete Identity component migration.
+    
+    !!! note
+        If you are migrating your user stores to the new user store managers with the unique ID capabilities, Follow the guidelines given in the [Migrating User Store Managers](https://is.docs.wso2.com/en/latest/setup/migrating-userstore-managers/) before moving to the next step
+                
+        ```tab="Linux / Mac OS"
```

Comment:
Move under step 9

Reply:
fixed in https://github.com/wso2/docs-apim/commit/d4a4773a2657877cd9ad081106301a4d29c1695c